src/coreclr/src/tools/aot/ILCompiler.Compiler/ILCompiler.Compiler.csproj

```diff
@@ -0,0 +1,455 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <AssemblyName>ILCompiler.Compiler</AssemblyName>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+
+    <!-- We're binplacing these into an existing publish layout so that F5 build in VS updates
+         the same bits tests expect to see in artifacts/crossgen2. That way we never need to wonder which
```

Comment:
Nit: This comment should not say artifacts/crossgen2

Reply:
Ah yeah, I copypasted the boilerplate.